jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -106,17 +132,17 @@ function main() {
   result=$(with_termination_log xmlstarlet sel -t -v /flow-build/result "$build_xml")
   if [[ $completed != "true" ]]; then
     echo "Pipeline not completed" | tee -a "${TERMINATION_LOG_PATH}" || true
-    exit "$jfr_rc"
+    terminate ${RESULT_ERROR_INFRA}
   fi
   if [[ ! $result ]]; then
     echo "No pipeline result in build.xml" | tee -a "${TERMINATION_LOG_PATH}" || true
-    exit "$jfr_rc"
+    terminate ${RESULT_ERROR_INFRA}
   fi
   echo "Pipeline completed with result: $result" | tee -a "${TERMINATION_LOG_PATH}" || true
   if [[ $result != "SUCCESS" ]]; then
-    exit 1
+    terminate ${RESULT_ERROR_CONTENT}
```

Comment:
I think in the outage situation we still would have had Jenkins logs saying connection to git-mirror failed.
The result probably would be ERROR_CONTENT, although we would like to see ERROR_INFRA in this specific case.

Reply:
I think in this case it would exit in line 81 during the loop of cloning the pipeline.
Error content is only returned if the pipeline completes.